website/docusaurus.config.js

```diff
@@ -14,6 +14,65 @@ module.exports = {
     events: require("./static/data/events.json"),
     repoUrl: "https://github.com/apache/apisix",
     docsUrl: "",
+    docs: [
+      {
+        name: "APISIX™",
+        nameInParamCase: "apisix",
+        description: "A dynamic, real-time, high-performance API gateway.",
+        shape: "triangle",
+        color: "var(--ifm-color-primary)",
```

Comment:
why color here is not a certain value

Reply:
It is used in [downloads](https://github.com/apache/apisix-website/blob/master/website/docusaurus.config.js#L23) so I left it as  it is.If the color must  be a certain value then I will change it:)

Reply:
No worries I'm just curious why is it not the same with others 🤣

Reply:
okay:)